nce/NCE-Pairwise-SM/overlap_features.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+import string
+import pickle
+from collections import defaultdict
+from argparse import ArgumentParser
+
+from nltk.stem.porter import PorterStemmer
+
+def load_data(dname):
+    stemmer = PorterStemmer()
+    qids, questions, answers, labels = [], [], [], []
+    print('Load folder ' + dname)
+    with open(dname+'a.toks', encoding='utf-8') as f:
```

Comment:
space around `+`

Reply:
Do you mean I need to add space for every `+`, right?

Reply:
yes, space around the operators

Reply:
I think you forgot to add space around operator (@rosequ's comment): `dname+'a.toks'` -> `dname + 'a.toks'`